orthoinference/src/main/java/org/reactome/orthoinference/StableIdentifierGenerator.java

```diff
@@ -70,4 +64,15 @@ public GKInstance generateOrthologousStableId(GKInstance inferredInst, GKInstanc
         inferredInst.addAttributeValue(stableIdentifier, orthoStableIdentifierInst);
         return inferredInst;
     }
+
+    private GKInstance createOrthologousStableIdentifierInstance(GKInstance stableIdentifierInst, GKInstance orthoStableIdentifierInst, String targetIdentifier) throws Exception {
+        orthoStableIdentifierInst = InstanceUtilities.createNewInferredGKInstance(stableIdentifierInst);
+        orthoStableIdentifierInst.addAttributeValue(identifier, targetIdentifier);
+        String identifierVersionNumber = "1";
+        orthoStableIdentifierInst.addAttributeValue(identifierVersion, identifierVersionNumber);
+        String orthoStableIdentifierName = targetIdentifier + "." + identifierVersionNumber;
+        orthoStableIdentifierInst.setDisplayName(orthoStableIdentifierName);
+        dba.storeInstance(orthoStableIdentifierInst);
```

Comment:
Maybe the storing of the new stable id instance can happen outside the method so there is no side effect to the creation and the method is only responsible for creating the Java object?